tests/extract_outline_test.py

```diff
@@ -0,0 +1,198 @@
+import os
+from pdfminer.pdfdocument import PDFDocument, PDFNoOutlines
+from pdfminer.pdfpage import PDFPage
+from pdfminer.pdfparser import PDFParser
+from tools import pdf2txt
+from tools import dumppdf
+
+introduction = """Introduction 
+Prices for academic journals, particularly science journals, have been on the rise 
+for decades.1 For researchers without access to an institutional library, these 
+cost increases have come to mean paying $30 or more for each journal article 
+they need to read. When they need to read hundreds of papers, these costs 
+quickly become unmanageable. In 2011, a researcher in Kazakhstan created a 
+website call Sci-Hub that allows anyone to access the full text of millions of 
+scientific articles for free. Librarians should be aware of Sci-Hub because many 
+of their patrons are already using it, and it will undoubtedly affect their expec-
+tations about what their library should provide and how quickly they should be 
+able to get articles. Librarians should also understand the risks that Sci-Hub can 
+pose to their patrons, their institutions, and themselves. 
+
+"""
+
+references = """References 
```

Comment:
Constants should be written in capital case (see [PEP-8 spec](https://peps.python.org/pep-0008/#constants)). Also, it would be useful to have a short description about what the constant is used for.